arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
  gains:
    joint1: {p: 100.0, i: 0.01, d: 10.0}
    joint2: {p: 100.0, i: 0.01, d: 10.0}
    joint3: {p: 100.0, i: 0.01, d: 10.0}
    joint4: {p: 100.0, i: 0.01, d: 10.0}
    joint5: {p: 100.0, i: 0.01, d: 10.0}
    joint6: {p: 100.0, i: 0.01, d: 10.0}
gripper_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - finger1_joint
  gains:
    finger1_joint:
      p: 100
      d: 1
      i: 1
      i_clamp: 1